packages/angular/src/app/shared/components/form-item-uploader/form-item-uploader.component.ts

```diff
@@ -0,0 +1,74 @@
+import { CommonModule } from '@angular/common';
+import {
+  Component, NgModule,
+} from '@angular/core';
+import { DxFileUploaderModule } from 'devextreme-angular/ui/file-uploader';
+
+@Component({
+  selector: 'form-item-uploader',
```

Comment:
may be better name is "form-file-uploader" ))
like form-texbox and form-datebox

Reply:
Rename to form-item-photo-upload, it's clearer, i think

Reply:
how about "form-photo-uploader"? it's simpler

Reply:
Then rename better form-item-photo => form-photo ?

Reply:
Fixed